### ISSUE TITLE
RC Test: John Travoltage 1.6.0-rc.2

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/qa#693]
@emily-phet , @terracoda , @KatieWoe  john-travoltage/1.6.0-rc.2 is ready for RC testing.
Document issues in https://github.com/phetsims/john-travoltage/issues and link to this issue.

This release includes PhET brand with accessibility features. It does not inclue PhET-iO.

This is a spot check RC following https://github.com/phetsims/qa/issues/676. Please verify that the issues listed in the sections below are resolved. Then some light testing to sanity check on a few platforms with a screen reader would be great.

Assigning to @kathy-phet and @KatieWoe for prioritization.

- [x] Tester = NS, Platform = Mac 11 + safari , Time = 1hr
- [x] Tester = KW, Platform = Win 10, Time = .5 hr

EDIT: JG moved these checkboxes here because they weren't clickable under sim links for some reason.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Test the Game Up harness on at least one platform.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL
spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test.
- When making an issue, check to see if it was in a previously published version.
- Try to include version numbers for browsers.
- If there is a console available, check for errors and include them in the Problem Description.
- As an RC begins and ends, check the sim repo. If there is a maintenance issue, check it and notify developers if there is a problem.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested,
list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>Issues to Verify</h3>

- [x] [Sim in the Android App](https://github.com/phetsims/john-travoltage/issues/443)
- [x] [Preferences Dialog button with JAWS](https://github.com/phetsims/john-travoltage/issues/442)
- [x] [Reading Blocks highlight in Safari](https://github.com/phetsims/john-travoltage/issues/435)
- [x] [Exposed HTML tag](https://github.com/phetsims/john-travoltage/issues/432)
- [x] [i18n issue](https://github.com/phetsims/john-travoltage/issues/431)
- [x] [Using VO keys to move arm/leg](https://github.com/phetsims/john-travoltage/issues/414)

These issues should have the "status:ready-for-qa" label. Check these issues off and close them if they are fixed.
Otherwise, post a comment in the issue saying that it wasn't fixed and link back to this issue. If the label is
"status:ready-for-review" or "status:fixed-pending-deploy" then assign back to the developer when done, even if fixed.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.2/phet/john-travoltage_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.2/phet/john-travoltage_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.2/phet/xhtml/john-travoltage_all.xhtml)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the accessibility-related feature that is being tested doesn't negatively affect the sim in any way. Here is a list of features to supported in this test: 
  - Alternative Input
  - Interactive Description
  - Sound and Sonification
  - Pan and Zoom
  - Mobile Description
  - Self Voicing
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).

<h3>Link(s)</h3>

- **[a11y View](link)**
- **[Simulation](all_phet_link)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to
screen readers. If you simply need a refresher on screen readers, please consult the
[QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md), which should have
all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the
a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and
descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [ ] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in
[this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a
screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a
screen reader.

<h3>Magnification</h3>

This sim supports magnification with pinch and drag gestures on touch screens, keyboard shortcuts, and mouse/wheel controls. Please test magnfication and make sure
it is working as intended and well with the use cases of the simulation. Due to the way screen readers handle user input, magnification is NOT expected to
work while using a screen reader so there is no need to test this case.

<h3>Final Requests</h3>

- [ ] If this sim is not in [this list](https://phet.colorado.edu/en/accessibility/prototypes) or up to date there, make an
issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign
to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>

  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary>

  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>

  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>

  We typically assign the developer who opened the issue in the QA repository.

  </details>

  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>

  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
QA is done

**Comment by jessegreenberg** [comment]
Thank you QA team, hopefully only one more spot check with the above issues resolved. Closing, with rc.3 coming shortly.
